.sonarqube/sonar-scanner.properties

```diff
@@ -14,8 +14,7 @@
 
 # Configuration file for Sonar Scanner used for CI 
 
-
-sonar.projectKey=usdot-fhwa-stol_autoware.auto
+sonar.projectKey=usdot-fhwa-stol_autoware-auto
```

Comment:
Asked these questions offline, but asking here again for others to see. 
Large portion of Sonarscan properties is commented off on both repositories. If it is commented off, how does sonar know to check which packages' unit test that it should check for coverage etc?